Eatery/Onboarding/Controllers/OnboardingViewController.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  OnboardingViewController.swift
+//  Eatery
+//
+//  Created by Reade Plunkett on 11/11/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import UIKit
+
+protocol OnboardingViewControllerDelegate {
+    func onboardingViewControllerDidTapNextButton(viewController: OnboardingViewController)
+}
+
+class OnboardingViewController: UIViewController {
+
+    internal var stackView: UIStackView!
+
+    internal var titleLabel: UILabel!
+    internal var subtitleLabel: UILabel!
+    internal var imageView: UIImageView!
+    internal var nextButton: UIButton!
```

Comment:
make these `private let`?

Reply:
Everything but stackView and subtitleLabel can be private. Since we access those two in OnboardingLoginViewController they have to be internal